### DISCUSSION TITLE
Added LevelClass Notice

### DISCUSSION THREAD

**Discussion by ljdj** [opcodesio/log-viewer#314] [Ideas]
Good morning,
I'm here to submit an idea, because I tried to make modifications, but without success.
In LogLevel of the LaravelLagLevel.php file, I wanted to do a self::Notice => LevelClass::notice(),
and put in ccss the Information for notice in blue and for info in green to differentiate the logs a little, could you either help make the modification or plan a code review.

thank you in advance, truthfully yours

**Comment by arukompas** [comment]
hey @ljdj , the `LevelClass` isn't extendable at the moment, as well as the colours set for the different log levels. You'll find it very hard to do it, because it's not developed with that in mind.

But I'll take this as a feature request - to be able to configure log level colours.

Hope this makes sense 😄

**Comment by ljdj** [comment]
Hey @arukompas ,
After some maneuvering of the files, I managed to put the notice in green after many hours of research!
It's not that I hate vue .js but bade.php may be preferable.

Consider putting notice on the logs and on the severity drop.

If you would like to see my modifications, I will share them with pleasure, but the public css will not be the best (adaptation to snatch.com)!

Thanks in advance

**Comment by ljdj** [comment]
PR validated by @arukompas thank you
